Add BMI160 temperature calibration instructions (#244)
SlimeVR/SlimeVR-Docs-Site: src/server/initial-setup.md

1. Now, set the IMU on a flat surface on each of the remaining 5 sides, until the serial console or led indicates that you can change position.
1. The IMU should now be calibrated and will begin to show rotation in the SlimeVR server.

Optionally, you can perform temperature calibration for BMI160 IMUs, which is a more advanced process but significantly reduces drift:

1. Place your trackers in a fridge or freezer for a period of time to cool them down to below 15°C. If you're unsure about the temperature of your trackers, you can check it in the SlimeVR Server when the tracker is turned on.

> 1. Turn on your trackers and gradually heat them. You can use something like a 3D printer bed, or a heat gun, but be careful not to overheat your battery or melt your case. The calibration is complete and automatically saved when the IMU reaches 45°C.

Reviewer: ```suggestion
1. Turn on your trackers and gradually heat them. You can use something like a 3D printer bed or a heat gun, but be careful not to overheat your battery or melt your case. The calibration will be complete and automatically saved once the IMU reaches 45°C.
```
